### ISSUE TITLE
Antialiased CNN

### ISSUE THREAD

**Issue by sidml** [researchmm/SiamDW#61]
Hi.
I was wondering if you tried using anti aliased cnn (https://github.com/adobe/antialiased-cnns). It improves the consistecy of CNN. It's a drop in replacement for maxpool and requires minimal changes to existing network. Inf fact, i replaced maxpool layer in your pretrained res22fc model with anti aliased version and it seems to improve performance on got 10k Validation set from 0.42 to 0.43.  The anti-aliased version doesn't add any new weights so it works. But, this is certainly not the proper way to evaluate the impact since the original network was trained under different setting.

I think if the corrected network is trained from scratch, it may give even better results. If someone decides to experiment with this, I hope they share the results. (sadly i don't have sufficient computational resources.)

Thanks.

**Comment by JudasDie** [comment]
> Hi.
> I was wondering if you tried using anti aliased cnn (https://github.com/adobe/antialiased-cnns). It improves the consistecy of CNN. It's a drop in replacement for maxpool and requires minimal changes to existing network. Inf fact, i replaced maxpool layer in your pretrained res22fc model with anti aliased version and it seems to improve performance on got 10k Validation set from 0.42 to 0.43. The anti-aliased version doesn't add any new weights so it works. But, this is certainly not the proper way to evaluate the impact since the original network was trained under different setting.
> 
> I think if the corrected network is trained from scratch, it may give even better results. If someone decides to experiment with this, I hope they share the results. (sadly i don't have sufficient computational resources.)
> 
> Thanks.

Hi, thanks for your share. I may try it later.
